src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -60,6 +60,7 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
                     country: billing.countryCode,
                 },
             },
+            paymentMethodConfiguration: { ...this._getDigitalRiverInitializeOptions().configuration.paymentMethodConfiguration },
```

Comment:
any reason why you are destructuring the object? Could this simply be 
```js
paymentMethodConfiguration: this._getDigitalRiverInitializeOptions().configuration.paymentMethodConfiguration,
```

or even better
```js
const { paymentMethodConfiguration } = this._getDigitalRiverInitializeOptions().configuration;

  // ...
  },
  paymentMethodConfiguration,
  onSuccess: (data?: OnSuccessResponse) => {
  // ...
```

Reply:
yeah looks better, thanks @capsula4